src/app/my-info/(account)/addresses/_components/Addresses/Address/Address.tsx

```diff
@@ -8,7 +8,7 @@ import { toast } from 'react-toastify';
 import { deleteAddress, putAddress } from '@/api/shippingAPI';
 import { Modal } from '@/components';
 import type { UserAddress } from '@/types/shippingType';
-import AddAddressModal from '../../AddAddresseModal/AddAddressModal';
+import AddAddressModal from '../../../../../../../components/AddAddresseModal/AddAddressModal';
```

Comment:
상대경로로 하니 너무 길게 나오는 것 같아요
```suggestion
import AddAddressModal from '@/components/AddAddresseModal/AddAddressModal';
```
아님 이 컴포넌트도 index에 넣는 건 어떨까요...?

Reply:
작업하면서 폴더 위치 옮긴거같은데 이렇게 되어있을줄 몰랐네요..
index로 넣을수 있는건 한번 싹 정리해보겠습니다